enaml/qt/qt_mpl_canvas.py

```diff
@@ -10,7 +10,10 @@
 from enaml.widgets.mpl_canvas import ProxyMPLCanvas
 
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg
-from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg
+try:
+    from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg
+except ImportError:
+    from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT
```

Comment:
I think this should be `backend_qt5agg`, where it actually resides:
https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/backends/backend_qt4.py#L35